src/components/Calculator/Calculator.js

```diff
@@ -74,7 +74,6 @@ export default function Calculator() {
       default:
         return;
     }
-
```

Comment:
## 1.
![image](https://user-images.githubusercontent.com/93039359/219301555-092a2426-496a-4f1d-968f-0fd8c0a76608.png)
## 2.
![image](https://user-images.githubusercontent.com/93039359/219301645-005c5a48-0760-4836-8359-95b77a3600eb.png)
## 3.
![image](https://user-images.githubusercontent.com/93039359/219301833-a9aa6deb-6a73-4a8c-adc6-454d9caec3e6.png)

- [ ] Kindly remove all these section descriptions because these are just guidelines for you and make your readme file not look clean and professional :thinking: Kindly remove them to make your readme file descriptive and professional :blush: